src/d3a/d3a_core/export.py

```diff
@@ -543,6 +557,9 @@ def _plot_ess_energy_trace(self, energy: dict, subdir: str, root_name: str):
         if len(data) == 0:
             return
         plot_dir = os.path.join(self.plot_dir, subdir)
+        # to do
+        if area.slug != "grid":
```

Comment:
This is not universal. The user can name their root areas whatever they want. Why do you want to omit adding the uuid for the root are in the first place?
If you realy want to do it, you could test for `area.parent == None`